keps/sig-network/0752-endpointslices/README.md

```diff
@@ -290,6 +285,8 @@ core/v1 Endpoints API is required. This allows the API to support services with
 no port remapping or all port services.
 
 ### Topology (Per Endpoint)
+**DEPRECATED** in v1.20 and removed in v1.21
```

Comment:
Can we entirely remove this field or do we have to leave it in v1beta1 indefinitely?